source/features/previous-version.tsx

```diff
@@ -1,41 +1,28 @@
 import React from 'dom-chef';
 import * as pageDetect from 'github-url-detection';
-
 import {VersionsIcon} from '@primer/octicons-react';
 
 import features from '../feature-manager.js';
 import observe from '../helpers/selector-observer.js';
 import api from '../github-helpers/api.js';
 import GitHubURL from '../github-helpers/github-url.js';
+import previousVersionQuery from './previous-version.gql';
 
-async function getPreviousCommitForFile(pathname: string): Promise<string> {
+async function getPreviousCommitForFile(pathname: string): Promise<string | undefined> {
```

Comment:
Actually we only needed `false` if the `webext-storage-cache` was involved, because `undefined === do not cache`

In this case there's no caching, so `undefined` is fine